### DISCUSSION TITLE
Advantage of qpkg over Docker Container

### DISCUSSION THREAD

**Discussion by InterN0te** [pdulvp/jellyfin-qnap#90] [General]
What are the advantages of using qpkg instead of Docker container ?

Is it more efficient for transcoding than Docker ?

**Comment by pdulvp** [comment]
Hi, 

Here, it's easy to install/update as it is installed through the App Center.
It a big 'zip' embedding all the dependencies for both jellyfin and ffmpeg, making it working on older NAS like mine. 
(libc is too old for instance in the QTS).

Not sure if the docker image embed all of dependencies or if it works on old NAS. (I don't use Docker nor Container Station).
Maybe easier in Docker for fine tuning for a specific setup; not sure though.

But for sure, it is not more efficient for transcoding. 
As the transcoding is done by gpu or cpu, it will depends on your NAS hardware, not the packaging of jellyfin docker or qkpg.

**Reply by InterN0te** [reply]
Thank you for your reply
